Feminine-Divine/periodo
# Created User Profile Page

#### Description
Issue FIXES #67




#### Describe the changes you've made

Created a profile page with 2 components - a details form  and  a profile card.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)

- No direct link ( in navbar or otherwise ) to navigate to Profile page currently.
- No backend created to store user details ( Information lost on hitting refresh )

#### Additional context (OPTIONAL)

![bandicam 2021-05-08 02-04-38-927](https://user-images.githubusercontent.com/73497800/117505822-4ad57d00-afa2-11eb-98b2-2c81f94669db.gif)

#### Test plan (OPTIONAL)

- Add '/profile' to the end of the URL to navigate to the Profile page
- Enter user details and profile picture in the form and click on the 'Submit' button
- Details should now appear to the right in the profile card.

#### Checklist

- [ ] I have added my name in the contributors list at the end of README.md file.
- [x] Open Source Program name - GSSoC'21
